### PR TITLE
Request for code Review

### PR DESCRIPTION
**Pull request by CornerstoneII**
# Request for code review.

> In this branch, we cloned an old version of the [Apple.com](https://rawcdn.githack.com/CornerstoneII/Building-with-Backgrounds-and-Gradients/0f9877fc7ef4aa2aeea2a6140a7ecc6545019669/index.html) page in line with the following requirements and features.

* Set background **image** on the **main** section page.
* Set a gradient color on the **nav** bar.
* Used **Font awesome 4 icons** and open-source images
* **image** sources from the [original](https://web.archive.org/web/20140301004610/http://www.apple.com/) page.
* **Flex** and **Floats** to achieve layouts.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="./images/favicon.png" type="image/png" sizes="16x16">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="index.css">
+    <title>Apple</title>
+</head>
+
+<body>
+    <main id="hero-image">
```
Comment (kubilaycaglayan) [inline index.html]: Unused id attributes.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            <ul class="flex">
+                <li><a href="#"><i class="fa fa-apple brand-icon"></i></a></li>
+                <li><a href="#">Store</a></li>
+                <li><a href="#">Mac</a></li>
+                <li><a href="#">iPod</a></li>
+                <li><a href="#">iPhone</a></li>
+                <li><a href="#">iPad</a></li>
+                <li><a href="#">iTunes</a></li>
+                <li><a href="#">Support</a></li>
+                <li class="search"><input type="search" name="search" id="q"></li>
```
Comment (kubilaycaglayan) [inline index.html]: Unused id attributes.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            </h1>
+            <h2 class="sub-header">
+                <a href="#">
+                    Everyone has something to share. See how people are using iPad in their unique ways to add to the
+                    world's story.
+                </a>
+            </h2>
+            <p class="description">
+                <a href="#" class="hover-underline">Watch the "Your verse" film and explore the stories</a>
+                <span id="play"><i class="fa fa-play-circle-o play-icon"></i></span>
```
Comment (kubilaycaglayan) [inline index.html]: Unused id attribute.

**Review by kubilaycaglayan — CHANGES_REQUESTED** [review]
Hello Team :wave: 
Status: Changes required :recycle: 

Amazing project! :boom:  I loved your precision :man_health_worker:  :woman_health_worker: 

Let's cover a few details and make sure that your project is at the top mastery level! :star: 

**Pull Request**
- [x] Pull Request has a meaningful title and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
Please work on the title :pray: 

**Repository**
- [x] Files not related to the project should not be committed and added to .gitignore (e.g. `.DS_Store`) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)) 
-> package-lock.json & package.json should be inside .gitignore file.
-> And please make sure you have deleted them from your GitHub repository too.
:notebook: **Note:** In the future projects(JavaScript section) we will not put these files into the .gitignore. But for now, it seems you are only using those files for local purposes.
:notebook: **Note 2:** Adding them into .gitignore may not be enough to delete them from the GitHub repository. Please research more to achieve the goal.

**HTML & CSS**
- [x]  You have unused ids, please delete them. Details are in the comments section. (We don't need those if we are not using them in CSS, as selectors.)

This is a great project, please keep up the good work. :chart_with_upwards_trend: 
You can reach out to me via Slack. :people_holding_hands: 
With all my respect...

File: README.md [hunk 4]
```diff
@@ -1 +1,52 @@
...
+
+## Built With
+- HTML5
+- CSS3
+- Font Awesome 4 Icons
+- SVG sources, images links in the original web page
+
+## Live Demo
+
+[Live Demo Link](https://rawcdn.githack.com/CornerstoneII/Building-with-Backgrounds-and-Gradients/0f9877fc7ef4aa2aeea2a6140a7ecc6545019669/index.html)
```
Comment (kitpao) [inline README.md:17]: - [ ] **[OPTIONAL]** Your live demo link is not updated. As a suggestion, so you don't have to be manually updating this link during your code reviews, use the second link (development link) in githack. That one will automatically load your last version in the same branch, so you don't have to worry about it later :)

**Review by kitpao — APPROVED** [review]
# 👋 Hello @CornerstoneII and @belsman 

💯 You did an amazing job! and also implemented all the previous TSE's requirements 🌟 Congratulations 😄 

**_Note:_** You have 1 optional suggestion. Optionals are **not required** in order to approve your project but are personal suggestions to improve your coding experience. Please check below to see it 🔽 

### 👏 Your project has been approved 👏
### 🚀 You may merge to master and submit the project completition form 🚀

___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Pull request merged**